runtime/dynstats.c

```diff
@@ -292,20 +315,82 @@ dynstats_readCallback(statsobj_t __attribute__((unused)) *ignore, void *b) {
 static rsRetVal
 dynstats_initNewBucketStats(dynstats_bucket_t *b) {
 	DEFiRet;
-	
+
 	CHKiRet(statsobj.Construct(&b->stats));
 	CHKiRet(statsobj.SetOrigin(b->stats, UCHAR_CONSTANT("dynstats.bucket")));
 	CHKiRet(statsobj.SetName(b->stats, b->name));
 	CHKiRet(statsobj.SetReportingNamespace(b->stats, UCHAR_CONSTANT("values")));
 	statsobj.SetReadNotifier(b->stats, dynstats_readCallback, b);
 	CHKiRet(statsobj.ConstructFinalize(b->stats));
-	
+
+finalize_it:
+	RETiRet;
+}
+
+/* try to open a file which has a state file. If the state file does not
+ * exist or cannot be read, an error is returned.
+ */
+static rsRetVal ATTR_NONNULL(1) openFileWithStateFile(dynstats_bucket_t *b) {
+	DEFiRet;
+	struct stat stat_buf;
+	uchar statefile[MAXFNAME];
+	uchar statefname[MAXFNAME];
+
+	uchar *const statefn = getStateFileName(b, statefile, sizeof(statefile));
```

Comment:
`statefn` sounds like `state-function` as opposed to `state-file-name`, how do you like `state_file_name` and `state_file_path` for `statefn` and `statefname respectively?

Reply:
yes, that sounds good.